packages/python-packages/api-stub-generator/apistub/_stub_generator.py

```diff
@@ -156,8 +156,18 @@ def _generate_tokens(self, pkg_root_path, package_name, version, namespace):
         apiview = ApiView(nodeindex, package_name, 0, version, namespace)
         modules = self._find_modules(pkg_root_path)
         logging.debug("Modules to generate tokens: {}".format(modules))
+
+        # find root module name
+        root_module = ""
+        if namespace:
+            root_module = namespace.split(".")[0]
+
         # load all modules and parse them recursively
         for m in modules:
+            if not m.startswith(root_module):
```

Comment:
Can you give me an example of this running for something we DONT own? Just for my own reference.

Reply:
Good question - I'm also curious :)

Reply:
It's not fully generalized. I have tried for few external packages and it didn't work. This fix is to mainly avoid other folders in package root other than "Azure". I will debug what's wrong in generating stub for external package.